cookbook/deployment/customizing_resources.py

```diff
@@ -48,21 +50,59 @@ def count_unique_numbers(x: typing.List[int]) -> int:
 
 
 # %%
-# Let's create a dummy task that determines the square of a number.
+# Define a task that computes the square of a number.
 @task
 def square(x: int) -> int:
     return x * x
 
 
 # %%
-# The tasks decorated with memory and storage hints can be used like regular tasks in a workflow.
+# The tasks decorated with memory and storage hints are used like regular tasks in a workflow.
```

Comment:
Does the "can be" phrase make more sense?

Reply:
Passive. Who or what are using the tasks with memory and storage hints?